ansible/roles/stack-sunbird/templates/sunbird_lms-service.env

```diff
@@ -164,4 +164,14 @@ exhaust_api_submit_endpoint=/request/submit
 exhaust_api_list_endpoint=/request/list/
 
 # Release-5.3.0 LR-324
-content_read_url=/content/v3/read/
\ No newline at end of file
+content_read_url=/content/v3/read/
+
+# Release-5.4.0 - LR-511
+sunbird_keyspace=sunbird
```

Comment:
@AmiableAnil is this variable referring SunbirdEd keyspace?

Reply:
Lern keyspace.